html-skeleton/index.html

```diff
@@ -1,9 +1,7 @@
 <!DOCTYPE HTML>
 <html>
-
-<head></head>
-<title>HTML Skeletons</title>
-
-<body>Hello, HTML!</body>
-
+  <head>
+    <title>HTML Skeletons</title>
+  <body>Hello, HTML!</body>
```

Comment:
The `body` element needs to be a sibling to `head`, not a child of it

Reply:
I have made the changes @codingcodymiller